examples/counterparts.rs

```diff
@@ -1,5 +1,6 @@
 use std::error::Error;
 
+// TODO what is this?
```

Comment:
This is just from the template to fill in having an example of the library :). https://doc.rust-lang.org/cargo/guide/project-layout.html